### PR TITLE
testing/botan: new aport

### PR DESCRIPTION
**Pull request by tcely**
https://botan.randombit.net/
Crypto and TLS for C++11

### CODE REVIEW

File: testing/botan/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+pkgname=botan
+_pkgreal=Botan
+pkgver=2.4.0
+pkgrel=0
+pkgdesc="Crypto and TLS for C++11"
+url="https://botan.randombit.net/"
+arch="x86_64 x86"
+license="BSD-2-Clause"
+depends=""
+depends_dev="python boost-dev libressl-dev sqlite-dev bzip2-dev zlib-dev"
```
Comment (jirutka) [inline testing/botan/APKBUILD]: There's no package named `python`, correct name is `python3` or `python2` for Legacy Python. The same for executable.

File: testing/botan/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		--with-sqlite3 \
+		--with-bzip2 \
+		--with-zlib
+	make
+}
+
+check() {
+	cd "$builddir"
+
+	LD_LIBRARY_PATH=. python src/python/botan2.py
```
Comment (jirutka) [inline testing/botan/APKBUILD]: Please fix `python` to `python2`.

File: testing/botan/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Contributor:
+# Maintainer:
```
Comment (jirutka) [inline testing/botan/APKBUILD]: Please fill your name and email at least to `Contributor`.

File: testing/botan/APKBUILD (hunk 2 above)
Comment (tcely) [reply testing/botan/APKBUILD]: Sure. Why prefer one symbolic link over the other?

Comment (jirutka) [reply testing/botan/APKBUILD]: `python` may be changed to python3 in the future. `python2` makes is more clear that it uses Legacy Python.

**Comment by tcely** [comment]
Done

**Pull request closed without merging**